smac/scenario/scenario.py

```diff
@@ -77,6 +77,10 @@ def __init__(self, scenario, cmd_args=None, run_id=1):
         self._groups = defaultdict(set)
         self._add_arguments()
 
+        # Make cutoff mandatory if run_obj is runtime
+        if 'run_obj' in scenario and scenario['run_obj'] == 'runtime':
```

Comment:
I assume that we always have 'run_obj' (as it is a required argument) and there is no need to check whether 'run_obj' exists.

But, as you check for it I wondered whether there is a case where we don't have 'run_obj'. If such a case would exist, we would need to handle it in this PR.  I assume there is no such case, so you can safely remove that check?